src/reactors/V3DutchOrderReactor.sol

```diff
@@ -103,6 +109,23 @@ contract V3DutchOrderReactor is BaseReactor {
         }
     }
 
+    function _updateWithGasAdjustment(V3DutchOrder memory order) internal view {
+        // positive means an increase in gas
+        int256 gasDeltaGwei = block.basefee.sub(order.gasSnapshot);
+
+        // Gas increase should increase input
+        int256 inputDelta = int256(order.baseInput.adjustmentPerGweiBasefee) * gasDeltaGwei / 1 gwei;
+        order.baseInput.startAmount =
+            order.baseInput.startAmount.boundedSub(0 - inputDelta, 0, order.baseInput.maxAmount);
```

Comment:
I would prefer doing boundedAdd vs. boundedSub, or doing add / sub and calling `bounded(min, max)` afterwards. Passing a negative value into a subtraction function feels like an antipattern